### PR TITLE
feat: add module loading

### PR DESCRIPTION
**Pull request by McDaddy**
## What this PR does / why we need it:
1. add module loading when dynamic module loading and prevent 404 page show up
2. move global-space to core
3. combine two event-hub into one
4. fix tooltop z-index issue in markdown editor
5. extract dynamic module registry to shell

## I have checked the following points:
- [x] I18n is finished and updated by cli
- [x] Form fields validation is added and length is limited
- [x] Display normally on small screen
- [x] Display normally when some data is empty or null
- [x] Display normally in english mode

## Does this PR introduce a user interface change?

❎ No

## ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |      add module loading        |
| 🇨🇳 中文    |  添加模块加载过渡            |

## Does this PR need be patched to older version?
❎ No

## Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

File: scheduler/src/proxy.ts [hunk 1]
```diff
@@ -24,7 +24,11 @@ const { BACKEND_URL, GITTAR_ADDR, UC_BACKEND_URL, ENTERPRISE_URL, FDP_URL } = en
 
 const API_URL = BACKEND_URL.startsWith('http') ? BACKEND_URL : `http://${BACKEND_URL}`;
 const UC_API_URL = UC_BACKEND_URL.startsWith('http') ? UC_BACKEND_URL : `http://${UC_BACKEND_URL}`;
-const ENTERPRISE_UI_URL = ENTERPRISE_URL?.startsWith('http') ? ENTERPRISE_URL : `http://${ENTERPRISE_URL}`;
+const ENTERPRISE_UI_URL = ENTERPRISE_URL
+  ? ENTERPRISE_URL.startsWith('http')
+    ? ENTERPRISE_URL
+    : `http://${ENTERPRISE_URL}`
+  : '';
```
Comment (daskyrk) [inline scheduler/src/proxy.ts:31]: Suggest add a new function to add http prefix.

File: core/src/index.tsx [hunk 2]
```diff
@@ -52,34 +46,39 @@ const matchEnterpriseRoute = () => {
   return target;
 };
 
-const App = () => {
+const App = ({ dynamicModules = [] }: { dynamicModules: DynamicModule[] }) => {
```
Comment (daskyrk) [inline core/src/index.tsx:49]: This is a optional prop?

File: core/webpack.config.js [hunk 3]
```diff
@@ -90,7 +90,6 @@ module.exports = () => {
     plugins: [
       new webpack.DefinePlugin({
         'process.env.NODE_ENV': JSON.stringify(nodeEnv),
-        'process.env.FOR_COMMUNITY': JSON.stringify(process.env.FOR_COMMUNITY),
```
Comment (daskyrk) [inline core/webpack.config.js:93]: This is still used in `App`

Comment (McDaddy) [reply core/webpack.config.js:93]: this `App` is shell App, not core

File: core/src/index.tsx (hunk 2 above)
Comment (McDaddy) [reply core/src/index.tsx:49]: yes. Mark it as optional

**Pull request merged**